profiler/test/Datadog.Profiler.IntegrationTests/LinuxOnly/WrapperLibraryTest.cs

```diff
@@ -80,55 +80,5 @@ public void EnsureAppDoesNotCrashIfProfilerDeactivateAndTracerActivated(string a
             var lines = File.ReadAllLines(logFile);
             lines.Should().ContainMatch(expectedErrorMessage);
         }
-
-        [TestAppFact("Samples.ExceptionGenerator")]
-        public void RedirectCrashHandler(string appName, string framework, string appAssembly)
-        {
-            var runner = new TestApplicationRunner(appName, framework, appAssembly, _output, enableTracer: true, commandLine: "--scenario 7");
-
-            runner.Environment.SetVariable("COMPlus_DbgMiniDumpType", string.Empty);
-
-            RegisterCrashHandler(runner);
-
-            using var processHelper = runner.LaunchProcess();
-
-            runner.WaitForExitOrCaptureDump(processHelper.Process, milliseconds: 30_000).Should().BeTrue();
-            processHelper.Drain();
-            processHelper.ErrorOutput.Should().Contain("Unhandled exception. System.InvalidOperationException: Task failed successfully");
-            processHelper.StandardOutput.Should().MatchRegex(@"createdump [\w\.\/]+createdump \d+")
-                .And.NotContain("Writing minidump");
-        }
-
-        [TestAppFact("Samples.ExceptionGenerator")]
-        public void DontRedirectCrashHandlerIfPathNotSet(string appName, string framework, string appAssembly)
```

Comment:
Maybe keep this test and rename it into GenerateDumpIfDbgRequested to validate that existing CLR behavior is still working with CrashTracking

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/5898/commits/904b7eb2a9d73f42be6febfb5d38ebde70b31878